api/documentation/src/pages/change-logs.md

```diff
@@ -8,11 +8,15 @@ title: Pass Culture API change logs
 💡 Important notice some old resources are going to be removed in the coming months.
 
 - If you were using `/v2/venue/<venue_id>/stocks` to manage stocks, you will have to migrate to this endpoint : [/public/offers/v1/products/ean](/rest-api#tag/Product-Offer-Bulk-Operations/operation/PostProductOfferByEan). The endpoint `/v2/stock` will not be available anymore starting from September, the 31st 2024.
-- If you were using `/v2/bookings` to manage bookings, you will have to migrate to those endpoints [/public/bookings/v1/bookings](/rest-api#tag/Bookings).  The endpoint `/v2/bookings` will not be available anymore starting from March, the 31st 2025.
+- If you were using `/v2/bookings` to manage bookings, you will have to migrate to those endpoints [/public/bookings/v1/bookings](/rest-api#tag/Bookings).  The endpoint `/v2/bookings` will not be available anymore starting from June, the 30th 2025.
```

Comment:
Je vois aussi un 31 septembre au-dessus, qui pourrait avantageusement être avancé au 30 :laughing:
Et ne plus être au futur : « is no longer available since September 30, 2024 ».

Style : 
```suggestion
- If you were using `/v2/bookings` to manage bookings, you will have to migrate to those endpoints [/public/bookings/v1/bookings](/rest-api#tag/Bookings).  The endpoint `/v2/bookings` will no longer be available starting from June 30, 2025.
```

Reply:
Ah oui le fameux 31 septembre 😅